README.rst

```diff
@@ -38,10 +38,12 @@ base code to provide times of the day in hebrew:
     סוף זמן תפילה גר"א - 10:28:00
     חצות היום - 12:40:00
     מנחה גדולה - 13:10:30
+    מנחה גדולה ל - 13:10:00
```

Comment:
Much better, please change in Hebrew to 
מנחה גדולה 30 דק

The current wording is not clear.
After that I will merge this PR.